SPECS/glusterfs/glusterfs.spec

```diff
@@ -574,55 +850,63 @@ sed -i -e 's/--quiet//' configure.ac
         %{?_without_fusermount} \
         %{?_without_georeplication} \
         %{?_without_ocf} \
-        %{?_without_rdma} \
         %{?_without_server} \
         %{?_without_syslog} \
         %{?_with_ipv6default} \
-        %{?_without_libtirpc}
-
-# fix hardening and remove rpath in shlibs
+        %{?_without_linux_io_uring} \
+        %{?_without_libtirpc} \
+        %{?_without_tcmalloc}
 
+## fix hardening and remove rpath in shlibs
+%if ( 0%{?fedora} && 0%{?fedora} > 17 ) || ( 0%{?rhel} && 0%{?rhel} > 6 ) || ( 0%{?mariner} && 0%{?mariner} >= 3 )
 sed -i 's| \\\$compiler_flags |&\\\$LDFLAGS |' libtool
-
+%endif
 sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|' libtool
 sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|' libtool
 
-gcc -v
-%make_build
+make %{?_smp_mflags}
```

Comment:
Why revert to the plain `make`? Wherever possible please keep `%make_build` as it includes any build flags we may wish to add or modify globally.

Reply:
all the changes here are upstream changes.  my goal was to align to upstream except when the mariner variable is used to define behavior specific to mariner.  i can do that here, but didn't know any better.

Reply:
modified per suggestion